projects/netgrif-components-core/src/lib/forms/email-submission/abstract-email-submission-form.component.ts

```diff
@@ -1,14 +1,14 @@
 import {EventEmitter, Input, OnDestroy, Output} from '@angular/core';
 import {FormBuilder, FormGroup, Validators} from '@angular/forms';
 import {FormSubmitEvent, HasForm} from '../has-form';
-import {Observable} from 'rxjs';
+import {LoadingEmitter} from '../../utility/loading-emitter';
 
 export abstract class AbstractEmailSubmissionFormComponent implements HasForm, OnDestroy {
 
     public rootFormGroup: FormGroup;
 
     @Input() public displayLegalNotice = true;
-    @Input() public loading = new Observable<boolean>();
+    public loading = new LoadingEmitter();
```

Comment:
LoadingEmitter is still a Subject internally and should therefore be destroyed with the component

Reply:
@minop corrected, completed in ngOnDestroy